packages/d/descope-core-js.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "descope-core-js-sdk",
+  "filename": "descope-core-js.min.js",
```

```suggestion
  "filename": "index.esm.js",
```
Looking at the package, your main file appears to be `index.esm.js`?